bin/compile

```diff
@@ -314,6 +297,35 @@ composer validate --no-check-publish --no-check-all --quiet "$COMPOSER" 2>/dev/n
 	EOF
 }
 
+# if prefer-stable is false and minimum-stability is not stable, warn about potential unstable platform installs
+[[ -f "$COMPOSER_LOCK" ]] && cat "$COMPOSER_LOCK" | python -c 'import sys, json; l = json.load(sys.stdin); sys.exit(l.get("minimum-stability", "stable") != "stable" and l.get("prefer-stable", False) == False);' 2> /dev/null || {
+	mcount "warnings.composer_lock.minimum_stability"
+	warning <<-EOF
+		Non-stable 'minimum-stability'!
+	
+		Your '$COMPOSER' contains a 'minimum-stability' lower than
```

Comment:
Would printing what they do have set for `minimum-stability` be useful here? Maybe show that snippet of the config?

Reply:
Done!